core/email-batches/comment-email-batch.php

```diff
@@ -529,10 +529,25 @@ protected function comment_children( $comment_id ) {
 	 */
 	protected function flood_controlled_recipient_ids() {
 
+		$approved_types = array(
+			'comment',
+		);
+
+		/**
+		 * Filter: replyable_comment_types
+		 *
+		 * Allow extra comment types to have moderation emails sent.
+		 *
+		 * @since 2.2.5
+		 *
+		 * @param array  Approved Comment Types.
+		 */
+		$approved_types = apply_filters( 'replyable_comment_types', $approved_types );
```

Comment:
For consistency with other filter names I'd suggest `prompt/comment_email_batch/approved_types` or `replyable/comment_email_batch/approved_types`.

Reply:
Hey Dylan, I changed the filter to your latter suggestion. @cyberhobo